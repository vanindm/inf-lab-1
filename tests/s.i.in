6 8
101262632 461683193 506295892 594990897 911365651 929418669 211012354 562616918 
892614081 999887835 779139455 431404519 958442633 405823669 594514458 946077192 
219867736 650293690 589131868 140674195 240001484 301778077 391753368 459707917 
317476423 85945414 67048923 999045590 661928078 6431455 903810124 164773384 
960378006 163710836 206424768 746231828 718316169 104264880 746394486 703571896 
472908368 640984417 914674745 415131883 115082135 494657569 746778687 749215094 
6 8
169655570 509674543 13091314 755901831 32117546 345848254 696134952 168568700 
280702885 58286295 169947566 229177939 807806477 642055310 126973098 513272844 
88395954 389895894 189083669 380728398 988801144 245444495 430522305 979287949 
276833566 716041266 910554965 835651356 285581358 603387223 84020945 752693245 
429292284 627650264 267986147 996034892 763343310 662804698 863490041 797082200 
727556858 920187819 167452852 719824841 713946472 526986497 423330983 753927003 
