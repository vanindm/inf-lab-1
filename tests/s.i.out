270918202 971357736 519387206 1350892728 943483197 1275266923 907147306 731185618 
1173316966 1058174130 949087021 660582458 1766249110 1047878979 721487556 1459350036 
308263690 1040189584 778215537 521402593 1228802628 547222572 822275673 1438995866 
594309989 801986680 977603888 1834696946 947509436 609818678 987831069 917466629 
1389670290 791361100 474410915 1742266720 1481659479 767069578 1609884527 1500654096 
1200465226 1561172236 1082127597 1134956724 829028607 1021644066 1170109670 1503142097 
